### ISSUE TITLE
Align on curl or wget for advanced installation

### ISSUE THREAD

**Issue by cjhenck** [Jigsaw-Code/outline-server#509]
While recently installing Outline with advanced mode, I had to install `curl` to run the script. Once I installed `curl`, the script then used `wget` to install docker. It would be great if we used only one of `curl` or `wget` so that fewer tools are needed on the host machine.

**Describe the solution you'd like**
I would recommend using wget in the Advanced Mode instructions, since Ubuntu and Debian have it by default.

**Comment by JonathanDCohen** [comment]
I've seen some misgivings already of people mistrusting `curl`'ing and executing a giant bash script to install Outline.  I'd argue that `wget` is even less trustworthy in this case because it does recursive downloads.  I'd also argue that anyone who has the ability to do advanced mode either already has or has the ability to acquire `curl` pretty easily

**Comment by cjhenck** [comment]
Well, `wget | bash` should have the same security properties as `curl | bash` since either way you're installing from a web script. Why force someone to install curl if wget is the same?

**Comment by besoeasy** [comment]
exactly, but this issue can be closed we have more important issues !
